dynamo/protocols/protocol_import_subtomos.py

```diff
@@ -156,3 +154,13 @@ def _getVolumeFileName(self, fileName, extension=None):
         else:
             baseFileName = "import_" + str(basename(fileName)).split(":")[0]
         return self._getExtraPath(baseFileName)
+
+    def _validate(self):
+        errors = []
+        if not list(self.iterFiles()):
```

Comment:
This could be a performance killer if we expect many files. for a few should be fine, but list() will actually have all files in it.

Reply:
I could change it to something like:
        try:
            next(self.iterFiles())
        except StopIteration:
            errors.append('No files matching the pattern %s were found.' % self.getPattern())
In this way, we only check the first element which is enough to know if the patter found a file or not.